src/Hidden/Hidden.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import {render} from '@testing-library/react'
+import {Hidden} from '.'
+import MatchMediaMock from 'jest-matchmedia-mock'
+import {behavesAsComponent, checkExports, renderStyles, checkStoriesForAxeViolations} from '../utils/testing'
+
+let matchmedia: MatchMediaMock
+describe('Hidden', () => {
+  beforeAll(() => {
+    matchmedia = new MatchMediaMock()
+  })
+  afterAll(() => {
+    matchmedia.clear()
+  })
+  behavesAsComponent({
+    Component: Hidden,
+    options: {skipAs: true, skipSx: true},
+    toRender: () => (
+      <Hidden on={'narrow'}>
+        <div>Hidden on narrow</div>
+      </Hidden>
+    ),
+  })
+  checkExports('Hidden', {
+    default: Hidden,
+    Hidden,
+  })
+  it('renders `on` prop as expected', () => {
+    const {container} = render(
+      <Hidden on={'narrow'}>
+        <div>Hidden on narrow</div>
+      </Hidden>,
+    )
+    expect(container).toMatchSnapshot()
+  })
+  it('renders the styles as expected when a single viewport value is provided as a string via `on` prop', () => {
+    const expectedStyles = {
+      [`@media screen and (min-width:768px)`]: {
```

Comment:
Ah yes, that makes perfect sense! When I wrote this, I was thinking "they should be hard-coded so that I can catch if the breakpoints fail too". However, now you said it and I realised that I am not testing the breakpoints here 😅 Thanks for pointing this out 🙌🏻 

In terms of where to drive them from, I used these values multiple times in `PageHeader` too and I always imported them from [useResponsiveHook](https://github.com/primer/react/blob/main/src/hooks/useResponsiveValue.ts#L9) . I updated the code to imported from `useResponsiveHook` again but this file might not be the best place now to keep them.  Would [theme-preval](https://github.com/primer/react/blob/main/src/theme-preval.js) make sense at all? If no, happy to move them over `src/utils/breakpoints.ts`!